src/connectors/playlistrandomizer.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+Connector.playerSelector = '.MuiList-root';
+Connector.videoTitleSelector = '.MuiListItemText-primary';
+Connector.trackArtSelector = '.Mui-selected .MuiListItemAvatar-root .MuiAvatar-circle .MuiAvatar-img';
+
+const trackSelector = '.Mui-selected .MuiListItemText-root .MuiListItemText-primary';
+
+Connector.getArtistTrack = () => {
+	let { artist, track } = Util.splitArtistTrack(Util.getTextFromSelectors(trackSelector), '-');
```

Comment:
Since every entry is a YouTube video title, it could be parsed by using the `Util.processYtVideoTitle` function.

Reply:
Thanks.